app/glossary.md

```diff
@@ -106,9 +115,18 @@ Teacher Analysis Division (TAD)
 Teaching School Alliance
 : Outstanding schools that work with others to provide high quality training to new and experienced staff. Cannot be lead schools or providers.
 
+Teacher Regulation Agency (TRA)
```

Comment:
Ordering should be alphabetical:
* Teacher Analysis Division (TAD)
* Teacher Qualifications Unit (TQU)
* Teacher Reference Number (TRN)
* Teacher Regulation Agency (TRA)
* Teacher supply model (TSM)
* Teaching School Alliance

Reply:
🤦‍♀️